library/core/class.dispatcher.php

```diff
@@ -796,6 +797,13 @@ private function _mapParts($Parts, $ControllerKey) {
             }
         }
     }
+
+    /**
+     * Trigger an event allowing addons to modify response headers.
+     */
+    public function sendHeaders() {
```

Comment:
Can this be private? We don't want to make functions public if there is no need.